spec/support/jbuilder_helper.rb

```diff
@@ -0,0 +1,11 @@
+# module JBuilderHelper
```

Comment:
Maybe we can delete this file if it's not used for now. We can always find it in the history when we need it.